src/Twig/Extension/FlashMessageExtension.php

```diff
@@ -11,58 +11,25 @@
 
 namespace Sonata\CoreBundle\Twig\Extension;
 
-use Sonata\CoreBundle\FlashMessage\FlashManager;
 use Twig\Extension\AbstractExtension;
 use Twig\TwigFunction;
 
 /**
  * This is the Sonata core flash message Twig extension.
  *
  * @author Vincent Composieux <composieux@ekino.com>
+ * @author Titouan Galopin <galopintitouan@gmail.com>
  */
 class FlashMessageExtension extends AbstractExtension
 {
-    /**
-     * @var FlashManager
-     */
-    protected $flashManager;
-
-    public function __construct(FlashManager $flashManager)
```

Comment:
this is a BC Break

Reply:
If I am correct, you can't delete construct and methods, you can change them to call methods from FlashMessageRuntime class, and add deprecations to construct and methods here.